onnxruntime/contrib_ops/cpu/transformers/beam_search_scorer.h

```diff
@@ -25,31 +25,28 @@ struct HypothesisScore {
 };
 
 struct BeamHypotheses {
-  void Init(const IGenerationParameters& parameters, gsl::span<HypothesisScore> beams);
-
-  // Number of hypotheses
-  int Size() const { return beams_used_; }
+  // As these are constructed as an unitialized array of memory,
```

Comment:
typo: uninitialized

Reply:
Locally fixed for the next test iteration